uui/src/lib/Drawer/Drawer.svelte

```diff
@@ -96,8 +97,17 @@
     padding-right: 1rem;
   }
 
-  // Target ListSubheader if nested in Drawer
+  // Override MDC targeting and target ListSubheader
   .drawer.mdc-drawer .mdc-deprecated-list-group__subheader {
-    color: var(--on-background);
+    color: rgba(255, 255, 255, 0.7);
```

Comment:
This has to be a css var to work on both themes. I believe it already exists in the palette.

Reply:
Also are we documenting these changes in Drawer? These really should just be website changes and not necessary involve the components

Reply:
I don't know if we should be hardcoding the width here as that is something the consumer should be doing. In the case of the drawer.

Reply:
I didn't see a corresponding color in the default palettes originally, although text-primary-on-background is close (.87 instead of .7 opacity). I guess i should check the hex colors again to make sure on of those isn't the correct one.

The thing i found was that mdc targets the subheader in the drawer and turns the color really dark ( rgba(0,0,0,.6) ) and that overrides whatever color the subheader is set to. I figured overriding their override as the default style shipped with the drawer component would be the right idea, but if you think that's the wrong direction i'm cool with yanking that and just overriding it for the website.

Yeah you're right about the width. I think i was just setting defaults because the original figma for Drawer shows 320 as well.